obinnaeye/quiz-generator
# # Pull Request For Addition Of difficulty level to quiz-generation-section UI

## Description 
This pull request introduces a new UI feature that allows users to select the difficulty level for the quiz they are generating through a dropdown menu. This enhancement aims to improve user experience by providing more control over the quiz content.

### Changes Made
- Implemented a dropdown menu for difficulty level selection in the quiz generation interface.
- Updated the UI components to accommodate the new feature.
- Added necessary styles to ensure the dropdown is visually integrated with the existing design.

### How to Test
1. Navigate to the quiz generation page in the application.
2. Verify that a dropdown menu for selecting the difficulty level is present.
3. Test the functionality by selecting different difficulty levels.

### Screenshots
![Screenshot from 2024-11-01 08-25-54](https://github.com/user-attachments/assets/ffd7769d-d1f9-41fe-97cb-12aca3b8275c)
